Features/Stories/Fieldtypes/AuthorsFieldType.feature

```diff
@@ -0,0 +1,96 @@
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I clicked on "Add another author" button
+        And I fill in "Author:" subform with:
+            | Name:   | Email:              |
+            | Author1 | author1@authors.pt  |
+            | Author2 | author2@authors.com |
+        When I click on the actionbar action "Publish"
+        Then I should see an element "Author" with value "Author1 <author1@authors.pt> Author2 <author2@authors.com>"
+
+    @javascript
+    Scenario Outline: Fill three authors then remove one then publish
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I clicked on "Add another author" button
+        And I clicked on "Add another author" button
+        And I fill in "Author:" subform with:
+            | Name:   | Email:              |
+            | Author1 | author1@authors.pt  |
+            | Author2 | author2@authors.com |
+            | Author3 | author3@authors.no  |
+        And I click on the "Remove this author" button number <index>
+        When I click on the actionbar action "Publish"
+        Then I should see an element "Author" with value "<value>"
+
+        Examples:
+            | index | value                                                      |
+            | 1     | Author2 <author2@authors.com> Author3 <author3@authors.no> |
+            | 2     | Author1 <author1@authors.pt> Author3 <author3@authors.no>  |
+            | 3     | Author1 <author1@authors.pt> Author2 <author2@authors.no>  |
+
+    @javascript
+    Scenario: Fill one author then remove it then publish
+        Given I am logged in as admin on PlatformUI
+        And a Content Type exists with identifier "Author" in Group with identifier "Content" with fields:
+            | identifier | Type     | Name   |
+            | name       | ezstring | Name   |
+            | author     | ezauthor | Author |
+        And I create a content of content type "Author" with:
+            | Name       |
+            | AuthorTest |
+        And I fill in "Author:" subform with:
+            | Name:   | Email:             |
+            | Author1 | author1@authors.pt |
+        And I clicked on "Add another author" button
+        And I click on the "Remove this author" button number "1"
+        When I click on the actionbar action "Publish"
+        Then I should see an element "Author" with value "This field is empty"
```

Comment:
> ignore me :)

you are not ignored :)

I see we had a small understanding, there is no field here of course as it is empty, so for now element, and if we find a way to introduce a abstract term for `Field` we can explore that as followup maybe? My concern on that is that Fields are vastly different, so if you try to hide that from the BDD for everything then we cause a lot of potential fragile magic to work behind the scenes. But anyway, give me a ping on what you guys agree on during today meeting.